src/components/Select/SelectOverlay.tsx

```diff
@@ -1,24 +1,39 @@
-import { FC, useMemo } from 'react';
+import { FC, useMemo, useState } from 'react';
 import { SelectProps } from './Select';
 import Select from 'react-select';
 import { SELECT_OVERLAY_STYLES } from './selectStyles';
 import theme from 'src/styles/theme';
-const SelectOverlay: FC<SelectProps> = ({ isSearchable = false, disabled, id, options, onChange, value, ...rest }) => {
+const SelectOverlay: FC<SelectProps> = ({ isSearchable = false, disabled, id, options, onChange, value, isMulti, ...rest }) => {
```

Comment:
and since you're already touching the `isSearchable`, could you change that as well? I realize it's a breaking change, but still. Maybe we should have a story to change all boolean properties to their names and drop the `is`...

Reply:
will write a story to complete this for all components